app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailFragment.kt

```diff
@@ -246,37 +188,38 @@ class EventDetailFragment : Fragment(R.layout.fragment_event_detail) {
                 header.editButton.isVisible = !isEditing
                 header.editButton.isEnabled = !isEditing
 
-                if (!isAttendee) {
-                    addTitleAndDoneButtonLayout.editTitleButton.isVisible = isEditing
-                    addTitleAndDoneButtonLayout.editTitleButton.isEnabled = isEditing
-                    addTitleAndDoneButtonLayout.titleTextView.isEnabled = isEditing
-
-                    addDescriptionLayout.editDescriptionButton.isVisible = isEditing
-                    addDescriptionLayout.editDescriptionButton.isEnabled = isEditing
-                    addDescriptionLayout.descriptionTextView.isEnabled = isEditing
-
-                    startTimeAndDateLayout.timeTextView.isEnabled = isEditing
-                    startTimeAndDateLayout.timeButton.isEnabled = isEditing
-                    startTimeAndDateLayout.timeButton.isVisible = isEditing
-                    startTimeAndDateLayout.dateTextView.isEnabled = isEditing
-                    startTimeAndDateLayout.dateButton.isEnabled = isEditing
-                    startTimeAndDateLayout.dateButton.isVisible = isEditing
-
-                    endTimeAndDateLayout.timeTextView.isEnabled = isEditing
-                    endTimeAndDateLayout.timeButton.isEnabled = isEditing
-                    endTimeAndDateLayout.timeButton.isVisible = isEditing
-                    endTimeAndDateLayout.dateTextView.isEnabled = isEditing
-                    endTimeAndDateLayout.dateButton.isEnabled = isEditing
-                    endTimeAndDateLayout.dateButton.isVisible = isEditing
-
-                    attendeesLayout.addAttendeeButton.isVisible = isEditing
-                }
-
                 reminderLayout.reminderTextView.isEnabled = isEditing
                 reminderLayout.reminderButton.isEnabled = isEditing
                 reminderLayout.reminderButton.isVisible = isEditing
             }
 
+            collectLatestLifecycleFlow(viewModel.isNotAttendeeAndIsEditing) {
```

Comment:
I'd usually rephrase such booleans to non-negated ones. In this case, how about `isCreatorEditing`? Creates less cognitive load